### PR TITLE
Nar'sie and Rat'var don't like each other

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Makes rat'var fight nar'sie. Improves support for cult and clockcult being run together.

## Why It's Good For The Game

Dynamic admin memes.
(Note: I am not suggesting in any way that you actually run this, it's a terrible idea but normal dynamic CAN trigger this if the multiple roundenders button is turned off (which I did all the time when I was admin))

## Changelog
:cl:
add: Ratvar now fights Narsie
add: Cult will not end the round on summon if clockcult exists.
tweak: Becoming a clockie counts towards the sacrifice objective.
fix: Report of clockcult shows 180 seconds instead of 1800 seconds
fix: Removes the delay between clockwork gateway opening and rat'var appearing
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Clockcult and Blood Cult inherently don't work together, while the Clockcult has to defend the ark for a set amount of time the Blood Cult can just summon Nar'Sie at any time they want since everyone else will be too occupied fighting on Rebee, additionally it can break Blood Cult if their sacrifice target just decides to stay on Rebee the whole round, I've had this happen when i ran Blood vs Clockcult once.

**Comment by PowerfulBacon** [comment]
That's why it's impossible in game, admisn are... well the big dum dum when it comes to adminbus and it's something everyone liked (spawning them both after round end).
This allows cult and clockcult to be somewhat doable if an admin fucks around with dynamic settings, not an actual thing.
If there is absolutely no admin intervention, the only thing this PR does is fix 2 minor bugs with clockcult.

**Comment by TeomanTheGreat** [comment]
Merge when

**Comment by Fronsis** [comment]
Speed Merge

not having the famous NT post-round end dialogue between them is not based.

**Comment by xboxnoob** [comment]
nice

File: code/modules/antagonists/clock_cult/clockwork_massive.dm [hunk 1]
```diff
@@ -215,20 +220,43 @@ GLOBAL_LIST_INIT(clockwork_portals, list())
...
+				SEND_SOUND(world, 'sound/magic/clockwork/ratvar_attack.ogg')
+				SpinAnimation(4, 0)
+				for(var/mob/living/M in GLOB.player_list)
+					shake_camera(M, 25, 6)
+					M.Knockdown(10)
+				if(prob(max(GLOB.servants_of_ratvar.len/2, 15)))
+					SEND_SOUND(world, 'sound/magic/demon_dies.ogg')
+					to_chat(world, "<span class='ratvar'>You were a fool for underestimating me...</span>")
+					qdel(ratvar_target)
+					for(var/datum/mind/M as anything in SSticker.mode?.cult)
```
Comment (ike709) [inline code/modules/antagonists/clock_cult/clockwork_massive.dm]:
```suggestion
					for(var/datum/mind/M as() in SSticker.mode?.cult)
```

File: code/modules/power/singularity/narsie.dm [hunk 2]
```diff
@@ -102,9 +104,30 @@
...
+				SpinAnimation(4, 0)
+				for(var/mob/living/M in GLOB.player_list)
+					shake_camera(M, 25, 6)
+					M.Knockdown(10)
+				if(prob(max(SSticker.mode?.cult.len/2, 15)))
+					SEND_SOUND(world, 'sound/magic/clockwork/anima_fragment_death.ogg')
+					SEND_SOUND(world, 'sound/effects/explosionfar.ogg')
+					to_chat(world, "<span class='narsie'>You really thought you could best me twice?</span>")
+					QDEL_NULL(clashing)
+					for(var/datum/mind/M as anything in GLOB.servants_of_ratvar)
```
Comment (ike709) [inline code/modules/power/singularity/narsie.dm]:
```suggestion
					for(var/datum/mind/M as() in GLOB.servants_of_ratvar)
```

**Comment by ike709** [comment]
YOLO

**Pull request merged**